src/AzureIoTHubClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Arduino. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <AzureIoTHubClient.h>
+
+Client* AzureIoTHubClient::sslClient = NULL;
+
+AzureIoTHubClient::AzureIoTHubClient(Client& sslClient)
+{
+    AzureIoTHubClient::sslClient = &sslClient;
+}
+
+void AzureIoTHubClient::begin()
+{
+#if defined(ARDUINO_ARCH_ESP8266)
+    Serial.setDebugOutput(true);
```

Comment:
This should probably stay in the example code. Developers may or may not want to see output, the library shouldn't force it.